### PR TITLE
AnimHost Training Pipeline - Phase 4: Expose more PAE, GNN hyperparameters

### PR DESCRIPTION
**Pull request by jasper2xF**
# Description
* Exposing pae epoch, gnn epoch and dropout to AniMHost config node. This is ~10 code lines per parameter without refactoring included in this pr.
* Replaced previous double/float spinbox input with line edit to avoid having to define precision explicitly
* Added documentation for adding hyperparams to DEV_GUIDE
* Unrelated TRAINING.md edit due to [PR 51](https://github.com/FilmakademieRnd/AnimHost/pull/51)

# Testing
* Test run with modified learning rate and dropout. Checked changes in the AI4Animation Network.py files
image

### CODE REVIEW

**Review by FaisalZ — APPROVED** [review]
All good!
